### PR TITLE
Athletes images can be stored in aws cloud service

### PR DESCRIPTION
**Pull request by larathordar**
#### Description
Changes proposed in this pull request:

* Athlete can have an image
* Athletes image is stored in a cloud service
#### What I have learned working on this feature: 
* To store photos on amazon AWS
* To setup active storage in a rails application

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,11 +3,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.3.4'
 gem 'active_model_serializers', '~> 0.10.0'
+gem 'aws-sdk-s3', require: false
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 gem 'rack-cors', require: 'rack/cors'
 gem 'rails', '~> 5.2.0.rc2'
+gem 'aws-sdk-s3', require: false
```
Comment (davidboll) [inline Gemfile]: Why is this added twice?

File: app/serializers/athlete/show_serializer.rb [hunk 2]
```diff
@@ -1,3 +1,13 @@
 class Athlete::ShowSerializer < ActiveModel::Serializer
-  attributes :name, :age, :home, :starttime
+  attributes :name, :age, :home, :starttime, :image
+
+#  def link
+#    api_v1_athlete_url(object)
+#  end
```
Comment (davidboll) [inline app/serializers/athlete/show_serializer.rb]: Are you using this? Why is it commented out?

File: spec/requests/api/v1/display_athletes_spec.rb [hunk 3]
```diff
@@ -15,5 +18,25 @@
...
+    #  before do
+    #    athlete.image.attach(io: File.open(fixture_path + '/dummy_image.jpg'), filename: 'attachment.jpg', content_type: 'image/jpg')
+    #  end
+
+  # it 'GET /api/v1/athletes' do
+  #     get "/api/v1/athletes/#{athlete.id}"
+  #     binding.pry
+  #    expect(object).to have_attribute(:image)
+  #  end
+#  end
```
Comment (davidboll) [inline spec/requests/api/v1/display_athletes_spec.rb]: Why are these commented out? :)

File: spec/requests/api/v1/display_athletes_spec.rb [hunk 4]
```diff
@@ -2,7 +2,10 @@
 RSpec.describe Api::V1::AthletesController, type: :request do
   describe 'GET /api/v1/athletes' do
     let!(:athlete) { create(:athlete) }
-    let(:object) { JSON.parse(response.body)}
+    let(:object) { JSON.parse(response.body) }
+    before do
+      #  athlete.image.attach(io: File.open(fixture_path + '/dummy_image.jpg'), filename: 'attachment.jpg', content_type: 'image/jpg')
```
Comment (davidboll) [inline spec/requests/api/v1/display_athletes_spec.rb]: Same here

File: app/serializers/athlete_serializer.rb [hunk 5]
```diff
@@ -1,8 +1,14 @@
...
   include Rails.application.routes.url_helpers
-  attributes :name, :link, :home
+  attributes :name, :link, :home, :image
 
   def link
     api_v1_athlete_url(object)
   end
+  def image
+    unless object.image.attachment.nil?
+      binding.pry
```
Comment (holgertidemand) [inline app/serializers/athlete_serializer.rb]: removes this

File: config/environments/development.rb [hunk 6]
```diff
@@ -17,7 +17,7 @@
     config.cache_store = :null_store
   end
 
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon
```
Comment (holgertidemand) [inline config/environments/development.rb:20]: have you added this to production.rb aswell?

**Review by holgertidemand — CHANGES_REQUESTED** [review]

File: config/environments/development.rb (hunk 6 above)
Comment (larathordar) [reply config/environments/development.rb:20]: done

**Review by holgertidemand — APPROVED** [review]
looks good

**Review by davidboll — APPROVED** [review]
LGTM 👍
